internal/config/config.go

```diff
@@ -1411,7 +1411,9 @@ func buildParser(name string, tbl *ast.Table) (parsers.Parser, error) {
 }
 
 func getParserConfig(name string, tbl *ast.Table) (*parsers.Config, error) {
-	c := &parsers.Config{}
+	c := &parsers.Config{
+		JSONStrict: true,
```

Comment:
I think @rawkode intended to have it default false to fix #6532, see https://github.com/influxdata/telegraf/issues/6532#issuecomment-542928948  Won't default true cause a regression there?

Reply:
The regression is in 1.13.0 because we changed the parse behavior, I disagree that this should parse be accepted by default because it has the non-obvious behavior of discarding the object.